README.md

```diff
@@ -1,24 +1,53 @@
-# README
+* ...
+ # Active_recod
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## About <a name = "About"></a>
+This project puts our knowledge of active record, models, and associations to the test. We built an application similar to Reddit (called Micro-Reddit) where a user can create a post and add comments to it.
 
-Things you may want to cover:
 
-* Ruby version
 
-* System dependencies
 
-* Configuration
+# RoR: micro_redit Application.
 
-* Database creation
 
-* Database initialization
+## About the project
+  First RoR project from the Microverse curriculum, implementation of the RoR framework to create a basic Active record, a muni 'redit'
 
-* How to run the test suite
+## 🔧 Built with<a name = "with"></a>
+  Ruby on Rails
 
-* Services (job queues, cache servers, search engines, etc.)
+  ## 🔨 Setup    <a name = "setup"></a>
+- Clone the project from github using
+- <code>$ https://github.com/Guy-Gustave/active_record/tree/feature_branch</code>
+- <code>cd active_record</code>
+- <code>$ bundle install</code>
+  
```

Comment:
# Changes required
- [ ] Kindly add instructions on how to do migrations upon clinging the project.